core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -620,12 +623,27 @@ public boolean isDateHeaderEnabled() {
 
     /**
      * Returns the {@link ExceptionHandler} that converts a {@link Throwable} to an
-     * {@link AggregatedHttpResponse}.
+     * {@link HttpResponse}.
      */
     public ExceptionHandler exceptionHandler() {
         return exceptionHandler;
     }
 
+    /**
+     * Returns the {@link ExceptionVerbosity} logged by services. The value of this property
+     * is one of the following:
+     * <ul>
+     *     <li>{@link ExceptionVerbosity#ALL} - logging all exceptions raised from services</li>
+     *     <li>{@link ExceptionVerbosity#UNHANDLED} - logging exceptions which are not handled by
+     *     {@link ExceptionHandler}s provided by a user and are not well-known exceptions
+     *     <li>{@link ExceptionVerbosity#NONE} - no logging exceptions</li>
+     * </ul>
+     * A log message would be written at {@code WARN} level.
```

Comment:
If the default behavior is `NONE`, should we need this feature at all? A user could specify his/her own `ExceptionHandler` that logs all or certain exceptions they are interested in, e.g.

```java
Server
  .builder()
  .exceptionHandler((ctx, cause) -> {
    // .. log ..
    return ExceptionHandler.ofDefault().convert(ctx, cause);
  })
```

Reply:
Nice suggestion. I guess most users prefer to use `LoggingService` and `ExceptionVerbosity` is seldom used. Let me remove `ExceptionVerbosity` and make it only working with annotated services for backward compatibility.

Reply:
Sounds good to me too.

Reply:
`ServerBuilder.exceptionVerbosity()` has been removed at https://github.com/line/armeria/pull/3674/commits/bd833a3acad30dcaa27df2bf0d4e5c00746d4c9b.